website/docs/r/lex_bot_intent.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "aws"
+page_title: "AWS: aws_lex_bot_intent"
+sidebar_current: "docs-aws-resource-lex-bot-intent"
+description: |-
+  Definition of an Amazon Lex Bot Intent used as an attribute in other Lex resources.
+---
+
+# aws_lex_bot_intent
+
+Identifies the specific version of an intent.
+
+## Example Usage
+
```

Comment:
From here onwards the page is for `aws_lex_bot`, s/be `aws_lex_intent`, all the args are different for an intent.

Reply:
This page is just for the intent block on a bot, which you are correct is different from an intent itself. I broke it out into it's own page to keep the aws_lex_bot page clean since bots contain a few nested resources. Would you suggest merging this page back into the main bot page?

Reply:
Ahh, I understand now. I would expect to see all args on the `aws_lex_bot` page, even the nested args, eg `environment` on https://www.terraform.io/docs/providers/aws/r/lambda_function.html

It's a shame they can't be referenced by arn or similar.

Reply:
Agreed. I am open to suggestions here. The reason i treated lex different than other resources is that it isn't a single level of nesting in many cases, for example

Bot
  AbortStatement
    Messages
      ...

That being said I see your point on this intent section and think it can be merged back into the main bot page. I will do that later today.

Reply:
Ok, I updated these docs to use the same formatting as the rest of the docs. And I put the attribute descriptions for all the nested attributes in each page they are used on instead of on separate pages. I think you were right it is easier to follow that way. Thanks for the input!